Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackForm.swift

```diff
@@ -346,9 +384,49 @@ class SentryUserFeedbackForm: UIViewController {
         button.setTitleColor(config.theme.submitForeground, for: .normal)
         button.addTarget(self, action: #selector(submitFeedbackButtonTapped), for: .touchUpInside)
         button.accessibilityIdentifier = "io.sentry.feedback.form.submit"
+        button.accessibilityHint = submitAccessibilityHint
         return button
     }()
     
+    lazy var submitAccessibilityHint = {
```

Comment:
`m`: I would prefer extracting this logic into an extra method or class and maybe detach it from UIKit dependencies and write plain Swift unit tests for it. I'm also fine with keeping dependencies to the textFields and imagView if we can easily write tests for it. I think this is a simple logic that can easily introduce bugs and testing all the edge cases with UI tests is too complicated IMO. Unit tests run wait faster and speed up development. Still having a few UI tests checking that the main use cases work fine, is even more important.

Reply:
I can see some benefit to it, although it will come with its own risk that the interface between the core controller and interface driver can contain its own kinds of bugs. 

I think we should at least wait until we've got everything working on a happy path we want to settle on, which it seems we're just about there, and maybe even not until we actually have some feedback and reported bugs to start writing unit tests for this. In any case a separate PR dedicated solely to that refactor I think would be best.

Reply:
> although it will come with its own risk that the interface between the core controller and interface driver can contain its own kinds of bugs.

That's why I think a combination of UI tests and unit tests is best. UI tests test the most critical happy path. Unit tests validate all the edge cases and maybe even the happy path.

> I think we should at least wait until we've got everything working on a happy path we want to settle on

The problem with that is that we potentially build code on untestable code, and quite often we forget refactoring it later. I would prefer fixing this right away. It's simply extracting the logic into an extra internal method. That should do the trick. But that's up to you.

Reply:
I'm tackling this in https://github.com/getsentry/sentry-cocoa/pull/4726